### PR TITLE
[th2-2334] prepare special build

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: conn-soup/README.md [hunk 1]
```diff
@@ -134,3 +134,10 @@ networkInterface: 0.0.0.0
 # Keeps all logical message parts together. If it set to value of `true` they won't be split to different batches
```
Comment (jean-michel-perli-sonsin) [inline conn-soup/README.md]:
```suggestion
# Maintains all the logical message parts together. If it is set to `true`, they won't be split into different batches
```

File: conn-soup/README.md [hunk 2]
```diff
@@ -134,3 +134,10 @@ networkInterface: 0.0.0.0
...
 evolutionSupportEnabled: true
 ```
+
+## Release notes
+
+### 3.6.0
+
++ Embedded service based on MINA decodes message as sender during send.
```
Comment (jean-michel-perli-sonsin) [inline conn-soup/README.md]:
```suggestion
+ Embedded service based on MINA decodes the message as sender during sending.
```

File: conn-soup/README.md [hunk 3]
```diff
@@ -134,3 +134,10 @@ networkInterface: 0.0.0.0
...
+  This approach is important for cases where pair of messages have the same protocol message type and different structures depending on direction.
```
Comment (jean-michel-perli-sonsin) [inline conn-soup/README.md]:
```suggestion
  This approach is important for cases in which a pair of messages have the same protocol message type and different structures depending on the direction.
```

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
Hello, please see if you agree with the changes proposed below:

conn-soup/README.md
Line 46 - Indicate whether the "connect" should automatically reconnect if **the** connection was closed by the opposite side.

Line 49 - Timeout in milliseconds before reconnecting if **the** connection was closed by the opposite side.

Line 52 - The required parameter for reconnection logic. If it **is** set **to `false`,** the reconnection won't **be performed**

Line 55 - **It enables** the feature to send heartbeat automatically. You can specify a time interval in 'heartbeatTimeout' option.

Line 58 - **It sends** Heartbeat every **second that is specified in this field** (if no other messages are sent during that time). This feature may be enabled by **setting** 'sendHeartBeats' option **to true**

Line 61 - **Maintains** all **the** logical message parts together. If it **is** set **to `true`,** they won't be split **into** different batches

Line 100 - Timeout in milliseconds before reconnecting if **the** connection was closed by the opposite side.

Line 103 - The required parameter for reconnection logic. If it **is** set **to `false`,** the reconnection won't **be performed**

Line 106 - **It enables** the feature to send heartbeat automatically. You can specify a time interval in 'heartbeatTimeout' option.

Line 109 - **It sends** Heartbeat **every second that is specified in this field** (if no other messages are sent during that time). This feature may be enabled by **setting** 'sendHeartBeats' option **to true**

Line 112 - **Maintains** all **the** logical message parts together. If it **is set to `true`,** they won't be split **into** different batches

Line 130 - If it is not set**,** the first **interface that is found** will be used.

In case of any question, please let me know.
Thanks

File: conn-soup/README.md [hunk 4]
```diff
@@ -134,3 +134,10 @@ networkInterface: 0.0.0.0
 # Keeps all logical message parts together. If it set to value of `true` they won't be split to different batches
 evolutionSupportEnabled: true
 ```
+
+## Release notes
+
+### 3.6.0
```
Comment (OptimumCode) [inline conn-soup/README.md:140]: I think we should add information about the new base image version we've started using for the conn boxes

**Review by OptimumCode — APPROVED** [review]

File: conn-soup/README.md (hunk 4 above)
Comment (Nikita-Smirnov-Exactpro) [reply conn-soup/README.md:140]: I will add it to the main README file, thanks

**Review by jean-michel-perli-sonsin — APPROVED** [review]
Please see if you agree with the changes below:

README.md
Line 34/39 - FIX connect rejects messages if **the** user tries to send them before the logon is **completed** (fixes issue with corrupting sender sequence number)

Line 47/52 - **An alert is sent** if **it gets an** ErrorMessage when sending raw message

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**